### PR TITLE
`Marketplace`: notify `Distributor` and `Vendor` of `Order`

### PR DESCRIPTION
**Pull request by zspencer**
https://github.com/zinc-collective/convene/issues/831

- [x] Include the Delivery Fee in the order email
- [x] Include the Delivery Address provided to Stripe
- [x] Include the Contact Info provided to Stripe
- [x] Mayhaps some tests?

### CODE REVIEW

File: app/furniture/marketplace/order.rb [hunk 1]
```diff
@@ -11,14 +11,22 @@ class Order < Record
...
       ordered_products.sum(0) do |ordered_product|
         ordered_product.product.price * ordered_product.quantity
       end
     end
+
+    delegate :delivery_fee, to: :marketplace
+
+    def price_total
+      product_total + delivery_fee
+    end
```
Comment (zspencer) [inline app/furniture/marketplace/order.rb:30]: Add some tests!

File: app/furniture/marketplace/stripe_events_controller.rb [hunk 2]
```diff
@@ -8,14 +8,30 @@ def create
...
+          [event.data.object.shipping.name,
+           shipping_address.line1,
+           shipping_address.line2,
+           "#{shipping_address.city}, #{shipping_address.state} #{shipping_address.postal_code}",
+        ].compact.join("\n")
+        order.update(delivery_address: delivery_address,
+          status: :paid,
+          contact_email: event.data.object.customer_details.email)
+
+        OrderReceivedMailer.notification(order).deliver_later
```
Comment (zspencer) [inline app/furniture/marketplace/stripe_events_controller.rb:31]: May want to expand the request specs here

File: app/furniture/marketplace/order.rb (hunk 1 above)
Comment (KellyAH) [reply app/furniture/marketplace/order.rb:30]: done!

Comment (zspencer) [reply app/furniture/marketplace/order.rb:31]: The tests look like they're covering `Cart#price_total` and `Cart#product_total`, not `Order#price_total` or `Order#product_total`

**Pull request merged**